src/frequenz/sdk/timeseries/_grid_frequency.py

```diff
@@ -89,15 +87,15 @@ def new_receiver(self) -> Receiver[Sample[Frequency]]:
         if not self._task:
             self._task = asyncio.create_task(self._send_request())
         else:
-            _logger.info("Grid frequency request already sent: %s", self._component)
+            _logger.info("Grid frequency request already sent: %s", self._component_id)
```

Comment:
Do we lose any information by logging only the ID instead of the whole object?

Reply:
Assuming it prints the type/category and given that not only a meter can send us frequencies, yes we do lose some information